examples/05-interoperability/05-converting-blocks-to-pdf/src/App.tsx

```diff
@@ -35,7 +36,13 @@ export default function App() {
 
   // Creates a new editor instance with some initial content.
   const editor = useCreateBlockNote({
-    schema: withMultiColumn(withPageBreak(BlockNoteSchema.create())),
+    schema: BlockNoteSchema.create().extend({
+      blockSpecs: {
+        pageBreak: createPageBreakBlockSpec(),
```

Comment:
Two things:
1. This seems inconsistend (why not `createColumnBlockSpec` or `PageBreak`) (see also my other comment)
2. `.extend` looks great. The only pro of `withMultiColumn` would be that you don't need to add both `ColumnBlock` and `ColumnListBlock`. Adding one of them would be invalid which makes it a bit weird that this is possible. Perhaps this is more the subject of a (tbd) extension API. Where 1 extension can register multiple block types it needs.

Reply:
Yea, I left `withMultiColumn` as an API. But this makes it clearer what is happening with a single API. I'll probably just use the withMultiColumn again, since that is easier for consumers